### PR TITLE
Merge master of all subtrees

### PR DESCRIPTION
**Pull request by arnout**
This merges the current master of all subtrees.

I'm not sure if it will be possible to merge it properly from github.
If not, we'll loose some history, so I'm going to go and force-push on
master in that case.

The following PRs are still missing:
~~https://github.com/prplfoundation/prplMesh-framework/pull/17~~
https://github.com/prplfoundation/prplMesh-framework/pull/12
https://github.com/prplfoundation/prplMesh-tools/pull/3
~~https://github.com/prplfoundation/prplMesh-controller/pull/6~~

My proposal is to rebase those on top of this commit. You can do such
a rebase with:

```
git fetch /path/to/your/local/component/repo branch-you-want-to-get
git cherry-pick -Xsubtree=<component> <base commit>..FETCH_HEAD
```
`<base_commit>`  is the sha1 of the base of the pull request (i.e. the master version on which the PR was based). If the PR contains only one commit, you can just use `FETCH_HEAD` instead of `<base_commit>..FETCH_HEAD`. Also, if you know that you need to cherry-pick two commits, you can use `FETCH_HEAD~2..FETCH_HEAD`.

### CODE REVIEW

**Review by tomereli — APPROVED** [review]
Approved - open PRs are a long way before we can merge them - so owners should close them and create new branches and PRs in the merged repos.

**Comment by arnout** [comment]
Merge button doesn't work properly, so I manually pushed to master and close now.

**Pull request closed without merging**
